TravelAgency-persistence/src/main/java/cz/muni/fi/travelAgency/dao/CustomerDaoImpl.java

```diff
@@ -0,0 +1,56 @@
+package cz.muni.fi.travelAgency.dao;
+
+import cz.muni.fi.travelAgency.entities.Customer;
+import org.springframework.stereotype.Repository;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import java.util.List;
+
+/**
+ * Implementation of {@link CustomerDao}
+ *
+ * @author Filip Cekovsky
+ */
+@Repository
+public class CustomerDaoImpl implements CustomerDao {
+
+    /**
+     * Entity manager for this class.
+     */
+    @PersistenceContext
+    private EntityManager manager;
+
+    @Override
+    public void create(Customer customer) {
+        manager.persist(customer);
+    }
+
+    @Override
+    public List<Customer> findAll() {
+        return manager.createQuery("select c from Customer c", Customer.class).getResultList();
```

Comment:
Queries in this class will not work because the table which stores Customer entities is called CUSTOMERS (check Customer entity).